Sources/Authenticator/Models/AuthenticatorStep.swift

```diff
@@ -58,6 +58,14 @@ public struct AuthenticatorStep: Equatable {
     /// so they are presented with the Confirm Sign In with MFA Selection View
     public static let continueSignInWithMFASelection = AuthenticatorStep("continueSignInWithMFASelection")
 
+    /// A user has successfully provided valid Sign In credentials but is required to select  a MFA type to continue
```

Comment:
It seems the docs comments were not updated properly to reflect what the new steps actually represent. 
The first one is about the MFA setup view, not selection view.
The second one is about Email MFA setup view, not TOPT setup view.